tests/test_utils.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import pytest
+
+from tensorflow.keras import losses as losses_module
+from tensorflow.keras import metrics as metrics_module
+
+from scikeras.utils import loss_name, metric_name
+
+
+class CustomLoss(losses_module.Loss):
+    def __init__(name, param):
```

Comment:
Typo? `def __init__(self, name, param)`

Reply:
Yep good catch.

Reply:
There's no real reason to even have this `__init__`...